### ISSUE TITLE
Error running angular UI on Module

### ISSUE THREAD

**Issue by valteres** [abpframework/abp#2994]
I have created  a new module with abp.io using the following commands:
abp new MyApp.MyModule -t module --tiered -ui angular --database-provider ef

When I try to run UI of MyApp.MyModule using "yarn start" in angular folder line I get the following error :
C:\***\MyApp.MyModule\angular>yarn start
yarn run v1.22.0
$ ng serve
An unhandled exception occurred: No projects support the 'serve' target.
See "C:\Users\***\angular-errors.log" for further details.
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

How can I run anguar UI of module (not app)?

**Comment by maliming** [comment]
Did you restore the package?

**Comment by valteres** [comment]
Yes. I have restored packages running yarn at the myModuleName\Angular folder.

In the angular.json there are two entries with module name under projects section (one for library and the other for application. - "projectType": "application", / "projectType": "library"). After renaming library one to a unique name by appending xxx, I get compile error for primeng :

ERROR in src/app/shared/shared.module.ts:6:29 - error TS2307: Cannot find module 'primeng/table'.
6 import { TableModule } from 'primeng/table';

I saw the primeng was removed from app template, but module template looks to be outdated / not fixed?

Are you using angular in a module or app ? (if I create app using abp it works. Only module does not)
